### DISCUSSION TITLE
HELP!! jQuery in Blocs Export not Functioning Properly!

### DISCUSSION THREAD

**Discussion by jsk5-web** [community/community#146950] [Programming Help]
### Body

TLDR: Designer taking on a developers role. Project’s jQuery (Isotope) functions fine in Blocsapp and when previewing in Chrome/Safari, but upon export the jQuery (Isotope) does not work.

Hello everyone!
I have been attempting to integrate Isotope into our Blocsapp website (this was not my choice to use) and everything functions fine in Blocsapp but when I export, Isotope does not function, it doesn’t filter. I can click the buttons and they respond by taking me back to the top of the page, but none of the images sort. As far as I know and can see everything is formatted correctly in Blocsapp. It is showing that the button is selected, but no response from the .item elements. Not too sure about the code though. This has been extremely frustrating and I am just trying to figure out how to get this to work.

I know I could just write the code outside of Blocs and put it in after export but I am trying to avoid doing more than I need to as I am a designer, not a developer and have already gone way beyond my abilities here. Any help would be greatly appreciated!!!

I dropped the code here if anyone is able to take a look at it, sorry it’s a mess and obviously some files are missing: [https://codepen.io/jsk5/pen/NPKRmGZ](url)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.
